api/v1beta1/nutanixmachine_types.go

```diff
@@ -115,6 +115,7 @@ type NutanixMachineStatus struct {
 	VmUUID string `json:"vmUUID,omitempty"`
 
 	// NodeRef is a reference to the corresponding workload cluster Node if it exists.
+	// Deprecated: Do not use. Will be removed in a future release.
```

Comment:
Can we remove the `NoreRef` column on line 142 since it will be empty?

Reply:
@dkoshkin Good catch, thank you.

Reply:
Fixed